assets/constants.js

```diff
@@ -271,11 +271,20 @@ var constants = {
   "SUPPORTED_AUDIO_LANGUAGES": [{
     "id": "en",
     "text": "English",
-    "related_languages": ["en"]
+    "related_languages": ["en"],
+    "speech_synthesis_code": "en-GB"
```

Comment:
I'm not sure the modelling here is right. I don't think we should have dicts with different keys inside a list that should be homogeneous.

Could we instead have two separate top-level constants: SUPPORTED_AUDIO_LANGUAGES and AUTOGENERATED_AUDIO_LANGUAGES, perhaps? The two things are handled rather differently anyway in e.g. the editor view, so I think it might make sense to separate them. Though you may need to unify both when generating the dropdown choices for the audio preferences.

Reply:
done